calico-enterprise_versioned_docs/version-3.19-2/release-notes/index.mdx

```diff
@@ -163,6 +163,8 @@ This release is supported for use in production.
 #### Updates
 
 * License usage data is now collected and stored locally in the cluster
+* Curator was removed from Calico Enterprise due to its limited reliability. To ensure that your Elasticsearch does not
+run out of space, please consult our documentation on [Data retention](../visibility/elastic/retention.mdx) and [Prometheus alerts for Elasticsearch](operations/monitor/metrics/elasticsearch-and-fluentd-metrics#create-prometheus-alerts-for-elasticsearch.mdx).
```

Comment:
```suggestion
run out of space, please consult our documentation on [Data retention](../visibility/elastic/retention.mdx) and [Prometheus alerts for Elasticsearch](../operations/monitor/metrics/elasticsearch-and-fluentd-metrics#create-prometheus-alerts-for-elasticsearch.mdx).
```

Reply:
This error is causing your build to fail.

Reply:
I'm not sure that anyone knows what Curator is. Probably better to reword this without using component names.